integration/raft/channel_participation_test.go

```diff
@@ -365,6 +369,124 @@ var _ = Describe("ChannelParticipation", func() {
 			channelInfo := channelparticipation.ListOne(network, orderer3, "participation-trophy")
 			Expect(channelInfo).To(Equal(expectedChannelInfoPT))
 		})
+
+		It("join application channel with join-block as follower via channel participation api", func() {
+			orderer1 := network.Orderer("orderer1")
+			orderer2 := network.Orderer("orderer2")
+			orderer3 := network.Orderer("orderer3")
+			orderers := []*nwo.Orderer{orderer1, orderer2}
+			members := []*nwo.Orderer{orderer1, orderer2}
+			peer := network.Peer("Org1", "peer0")
+
+			By("starting two orderers")
+			for _, o := range orderers {
+				startOrderer(o)
+				channelparticipation.List(network, o, nil)
+			}
+
+			genesisBlock := applicationChannelGenesisBlock(network, members, peer, "participation-trophy")
+			expectedChannelInfoPT := channelparticipation.ChannelInfo{
+				Name:            "participation-trophy",
+				URL:             "/participation/v1/channels/participation-trophy",
+				Status:          "active",
+				ClusterRelation: "member",
+				Height:          1,
+			}
+
+			for _, o := range members {
+				By("joining " + o.Name + " to channel as a member")
+				channelparticipation.Join(network, o, "participation-trophy", genesisBlock, expectedChannelInfoPT)
+				channelInfo := channelparticipation.ListOne(network, o, "participation-trophy")
+				Expect(channelInfo).To(Equal(expectedChannelInfoPT))
+			}
+
+			By("waiting for the leader to be ready")
+			memberRunners := []*ginkgomon.Runner{ordererRunners[0], ordererRunners[1]}
+			leader := findLeader(memberRunners)
```

Comment:
I wounder how long does it take to converge with leadership selection having only two active replicas.

Reply:
**blinks**